app/enterprise/2.2.x/immunity/install-configure.md

```diff
@@ -25,32 +25,17 @@ Install Immunity by downloading, installing and starting the Collector App on Do
 ### Prerequisites
 To complete this installation you will need:
 
-* A valid *Bintray* account. You will need your *username*, account *password* and account *API Key*.
-  - Example:
-    * Bintray Access key = `john-company`
-    * Bintray username = `john-company@kong`
-    * Bintray password = `12345678`
-    * Bintray API key = `12234e314356291a2b11058591bba195830`
-  - The API Key can be obtained by visiting [https://bintray.com/profile/edit](https://bintray.com/profile/edit) and selecting *API Key*
-
 * A Docker-enabled system with proper Docker access.
 
 * Kong Enterprise 2.2.x or later is installed on Docker.
 
 * A valid Kong Enterprise License JSON file, including a license for Immunity.
```

Comment:
```suggestion
* A valid [Kong Enterprise License](/enterprise/{{page.kong_version}}/deployment/access-license/) JSON file, including a license for Immunity.
```
Might be worth linking to the license doc, but I think this is sufficient otherwise.